### PR TITLE
README update

### PR DESCRIPTION
**Pull request by ia**
* **Please check if the PR fulfills these requirements**
- [x] The changes have been tested locally
- [x] There are no breaking changes

* **What kind of change does this PR introduce?**
README.md update.

* **What is the current behavior?**
There is no direct link to supported hardware table.

* **What is the new behavior (if this is a feature change)?**
Add _Supported Hardware_ markdown-powered subtitle, so HTML anchor could be used to provide the direct link to the related information [like this](https://github.com/ia/IronOSf/tree/readme#supported-hardware)!

* **Other information**:

Hello. Long time no see. As you may noticed, in the discussions or during online conversations new people are keep asking from time to time the same old question: "what hardware are supported?". With this update, table will be available via github interface and on the read-the-docs instance of the documentation through the direct link (instead of giving people instructions like "open this link **first** and **then** scroll down to the table). So I think this can be pretty useful in the future to share with new users or just for those who want to check the current status of supported hardware in one direct click.

Plus, a bit of actualization here and there, since we're not ~~in Kansas anymore~~ dealing with _**irons**_ only anymore. ;)

However, since I decided to edit README anyway, I took some liberty to polish & beautify it a bit and make some sections more _structural_ (like buttons control). If I took _too much_ liberty then just let me know, because as always I'm very curious about the feedback.

Oh, and I know that "licence" is grammatically fully correct but I just thought to use more commonly wide "license". And of course let me know if I did make any mistypes/mistakes myself.

P.S. I'm sorry for the absence but I tried to follow up & check up with the project. Just had very busy a couple of months. But I always sincerely happy to be around & being useful in any way for this project. :pray:

### CODE REVIEW

**Review by Ralim — APPROVED** [review]

**Comment by Ralim** [comment]
Thank you 😍

I've been busy this year as well so it hasn't moved as fast anyway

File: README.md [hunk 1]
```diff
@@ -3,23 +3,25 @@
...
 
-The firmware implements all of the standard features of a 'smart' soldering iron, with lots of little extras and tweaks.
-I highly recommend reading the installation guide fully when installing on your iron. And after install just explore the settings menu.
+The firmware implements all of the standard features of a _smart_ soldering hardware, with lots of little extras and tweaks.
+I highly recommend reading the installation guide fully when installing on your device. And after install just explore the settings menu.
 
-For soldering irons that are designed to be powered by 'smart' power sources (PD and QC), the firmware supports settings around the negotiated power and voltage.
-For soldering irons that are designed to be powered by batteries (TS100 & Pinecil), settings for a cutoff voltage for battery protection are supported.
+For soldering hardware that are designed to be powered by _smart_ power sources such as _PD_ or _QC_, the firmware supports settings around the negotiated power and voltage.
+For soldering hardware that are designed to be powered by batteries (_TS100_ & _Pinecil_), settings for a cutoff voltage for battery protection are supported.
```
Comment (yo3fxy) [inline README.md:16]: typo : “soldering irons are” vs. “soldering hardware is”

File: README.md [hunk 2]
```diff
@@ -64,47 +64,65 @@ For notes on installation for your device, please refer to the flashing guide fo
...
+- in _soldering mode_:
+  - short press of `+/A` / `-/B` buttons changes the soldering temperature;
+  - long press of the `+/A` button enables _boost mode_ (increasing soldering temperature to the adjustable setting as long as the button is pressed);
+  - long press of the `-/B` button enters _standby mode_ and stops heating;
+- in _standby mode_:
+  - long press of the `+/A` button enters _soldering temperature adjust mode_ (the same as the one in the _soldering mode_, but allows to adjust the temperature before heating up);
+  - long hold of the `-/B` button enters the [_debug menu_](https://ralim.github.io/IronOS/DebugMenu/);
+- in _menu mode_ (to make it short here):
+  - `-/B` scrolls & cycles through menus and submenus;
+  - `-/A` enters to menu & submenu settings or changes their values if they are activated already.
```
Comment (yo3fxy) [inline README.md:117]: typo : “+/A” not “-/A”

File: README.md (hunk 1 above)
Comment (discip) [reply README.md:16]: @yo3fxy
Please create a PR to fix this. 😊

File: README.md (hunk 2 above)
Comment (discip) [reply README.md:117]: same

**Comment by ia** [comment]
Dear @yo3fxy! Thank you for pointing out these mistypes, much appreciated. The more I edited text the less I saw any particular letters. :|

Good job. I've made a patch for this patch already taking into consideration your feedback. But could you, please, double check it again here (#1996) after me as well to make sure we didn't miss anything else? Thanks.

**Pull request merged**